aggregator/aggregator.yml

```diff
@@ -0,0 +1,5 @@
+tagline: Wait for a sequence of messages and combine them
+desc: Aggregator patiently collects a sequence of messages and combine them once all have been received.
+helps: Ballerina provides convenient map and table data structures for temporary message storage. 
```

Comment:
Mention we also have libraries for Redis for distributed storage and database support for permanent storage.

Reply:
also mention our cache library, when users need control of automatic clean up.